[2021-12-14 07:50:16,675] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-14T07:50:15.180559+00:00 [queued]>
[2021-12-14 07:50:16,687] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-14T07:50:15.180559+00:00 [queued]>
[2021-12-14 07:50:16,687] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-14 07:50:16,688] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-14 07:50:16,688] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-14 07:50:16,700] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): create_pet_table> on 2021-12-14 07:50:15.180559+00:00
[2021-12-14 07:50:16,705] {standard_task_runner.py:52} INFO - Started process 138 to run task
[2021-12-14 07:50:16,710] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag', 'create_pet_table', 'manual__2021-12-14T07:50:15.180559+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/pets_dag.py', '--cfg-path', '/tmp/tmpfsswsrgx', '--error-file', '/tmp/tmpv8fr34nm']
[2021-12-14 07:50:16,710] {standard_task_runner.py:77} INFO - Job 12: Subtask create_pet_table
[2021-12-14 07:50:16,755] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-14T07:50:15.180559+00:00 [running]> on host 4ee0ea791fc7
[2021-12-14 07:50:16,809] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_operator_dag
AIRFLOW_CTX_TASK_ID=create_pet_table
AIRFLOW_CTX_EXECUTION_DATE=2021-12-14T07:50:15.180559+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-14T07:50:15.180559+00:00
[2021-12-14 07:50:16,820] {base.py:79} INFO - Using connection to: id: ***_conn_id. Host: postgres-***, Port: 5432, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-14 07:50:16,825] {dbapi.py:225} INFO - Running statement: -- create hello_world_table
DROP TABLE IF EXISTS pet;
CREATE TABLE pet (
    pet_id SERIAL PRIMARY KEY,
    name VARCHAR NOT NULL,
    pet_type VARCHAR NOT NULL,
    birth_date DATE NOT NULL,
    OWNER VARCHAR NOT NULL
    );, parameters: None
[2021-12-14 07:50:16,860] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag, task_id=create_pet_table, execution_date=20211214T075015, start_date=20211214T075016, end_date=20211214T075016
[2021-12-14 07:50:16,881] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-14 07:50:16,924] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
